tools/tslint-rules/noExposedTodoRule.js

```diff
@@ -0,0 +1,34 @@
+const ts = require('typescript');
```

Comment:
We should compare the required changes in the build package (from Material2 -> Flex-Layout). These changes will help us understand what stills needs to be configured for a generic build package.